include/swift/AST/DiagnosticsSema.def

```diff
@@ -436,6 +436,9 @@ ERROR(unspaced_unary_operator,sema_nb,none,
 
 ERROR(use_unresolved_identifier,sema_nb,none,
       "use of unresolved %select{identifier|operator}1 %0", (Identifier, bool))
+WARNING(confusable_character,sema_nb,none,
```

Comment:
Oh, and it should also be a NOTE rather than a WARNING because it should be attached to the previous error.